src/components/Spinner/_Spinner.scss

```diff
@@ -0,0 +1,29 @@
+@import '../../../styles/mixins';
+@import '../../../styles/variables';
+
+.spinner-on {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    z-index: 10;
+    height: 100%;
+    width: 100%;
+    background-color: $transparent-black;
+    position: fixed;
+    .spinner {
+        border: 16px solid #f3f3f3; /* Light grey */
```

Comment:
these colors should be variables, also blue doesn't match our theme, it should be yellow like or gray

Reply:
Done!